cmd/duffle/bundle_show.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func newBundleShowCmd(w io.Writer) *cobra.Command {
+	bsc := &bundleShowCmd{}
+	bsc.w = w
+
+	const usage = ` Returns information about an application bundle.
```

Comment:
nit: I don't really like that we duplicate the usage - whenever we'll modify something, we're going to have to modify it twice.

Could you please add this as a `const` outside the function and reference it in `show.go`?

Reply:
Move the shared long-form usage into a method here: https://github.com/deislabs/duffle/pull/631/files#diff-67b87ff356fa854fe2a2ca92f31e2c18R41.

With this you can indicate if the usage should show examples of the form `duffle show` or `duffle bundle show`.

Reply:
Nice!